events/code_review_events/workflow.py

```diff
@@ -216,78 +252,145 @@ def publish_results(self, payload):
 
         return True
 
-    def parse_pulse(self, payload):
-        assert self.publish is True, "Publication disabled"
-
-    async def start_risk_analysis(self, build):
-        """
-        Run risk analysis by triggering a Taskcluster hook
-        """
-        assert isinstance(build, PhabricatorBuild)
-        assert build.state == PhabricatorBuildState.Public
-        try:
-            if self.should_run_risk_analysis(build):
-                task = self.community_hooks.triggerHook(
-                    "project-relman",
-                    "bugbug-classify-patch",
-                    {"DIFF_ID": build.diff_id},
-                )
-                task_id = task["status"]["taskId"]
-                logger.info("Triggered a new risk analysis task", id=task_id)
+    def add_new_jobs(self, revision, selected_tasks):
```

Comment:
I would be more comfortable with that part outside the main workflow (it's starting to grow outside code review)